### PR TITLE
chore(issues):add github issue templates

### PR DESCRIPTION
**Pull request by steven-zou**
Signed-off-by: Steven Zou <szou@vmware.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,8 +88,8 @@ Now, this project is maintained and developed by the [Harbor operator workgroup]
 
 ## Community
 
-* Slack channel `#harbor-operator-dev` at [CNCF Workspace](https://slack.cncf.io)
```
Comment (holyhope) [inline README.md:91]: [From CI](https://github.com/goharbor/harbor-operator/runs/594276794), please keep dash for unordered list:
```
README.md: 91: MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md: 92: MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+**Steps to reproduce the problem:**
+Please provide the steps to reproduce this problem.
+
+**Versions:**
+Please specify the versions of following systems.
+
+- harbor operator version: [x.y.z]
+- harbor version: [x.y.z]
+- kubernetes version: [x.y.z]
+
```
Comment (holyhope) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]:
```suggestion
- Any additional relevant version such as CertManager

```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+**Additional context:**
+
+- **Harbor dependent services:**
+  - Context info of postgreSQL
+  - Context info of Redis
+  - Context info of storage
+- **Log files:** Collect logs and attach them here if have.
```
Comment (holyhope) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]:
```suggestion
- **Log files:** Collect logs and attach them here if have.
- **Kubernetes:** How Kubernetes access was provided (what cloud provider, service-account configuration, ...)
```

File: README.md (hunk 1 above)
Comment (steven-zou) [reply README.md:91]: This one has been updated in PR.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (steven-zou) [reply .github/ISSUE_TEMPLATE/bug_report.md:20]: ->
Any additional relevant versions such as CertManager

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (steven-zou) [reply .github/ISSUE_TEMPLATE/bug_report.md:28]: added

**Pull request merged**